system/metrics/sinks/prometheus/prometheusSink.cpp

```diff
@@ -180,6 +180,9 @@ void PrometheusMetricSink::toPrometheusHistogram(const std::string &name, const
 std::string PrometheusMetricSink::getPrometheusMetricName(const std::shared_ptr<IMetric> &pMetric)
 {
     std::string name = pMetric->queryName();
+    // Convert to lower case - per promtool output: "metric names should be written in 'snake_case' not 'camelCase'"
```

Comment:
Perhaps out of scope of this issue, but lower casing all metric names per Prometheus promtool output:
dfs_8880_WsDfs_KeepAlive_seconds metric names should be written in 'snake_case' not 'camelCase'
dfs_8880_WsDfs_KeepAlive_seconds no help text